Formula/ffmpeg.rb

```diff
@@ -67,9 +70,11 @@ def install
       --enable-libass
       --enable-libopencore-amrnb
       --enable-libopencore-amrwb
+      --enable-libopenjpeg
       --enable-librtmp
       --enable-libspeex
       --enable-videotoolbox
+      --disable-decoder=jpeg2000
```

Comment:
I don't have a problem with enabling openjpeg over it, but is it possible to compile with both?

Reply:
@slhck I have re-enabled the bad build-in coder, but I’m personally not convinced that it’s a good idea to use it.

Reply:
If you don't want to use it, it's better to override it than to manually disable a default that other users may be expecting to be present.

I am aware of the fact that some users might not know and that the default could be better, but it's similar for other codecs where more encoders are available, such as AAC, where the built-in is still the default.

Reply:
@slhck FYI: the build-in AAC is now very good, and we actually recommend to use it.